index.bs

```diff
@@ -675,8 +675,9 @@ Tooling for Discussions and Publications</h4>
 			for producing its documentation and deliverables
 			or for official group discussions
 			<em class=rfc2119>should</em> be usable
```

Comment:
I agree with the original comment in #537 that if you don't call this out more explicitly, there will be problems. My experience is that while in many cases staff are alert to the issue, they are easily put under pressure by Working Groups who are keener to get their work finished than to be inclusive, and it is important to draw fairly bright red lines. In part this is because even with goodwill and effort, it is often dificult to find accessible tooling - many of the cpmmon tools people know from their work are in fact seriously problematic :(

Reply:
I think you missed the other half of the edit on line 680?